scripts/init-vagrant-minion.sh

```diff
@@ -16,13 +16,15 @@ if [ ! -d /vagrant/cloned-projects/builder-base-formula/.git ]; then
     git clone ssh://git@github.com/elifesciences/builder-base-formula /vagrant/cloned-projects/builder-base-formula
 fi
 
-# project's `salt` file is mounted at `/srv/salt/` within the guest
+# project's `salt` file is mounted at `/project` and symlinked at `/srv/salt`
+# and `/srv/pillar` within the guest
 # by default the project's top.sls and pillar data is disabled by file naming.
 # hook that up now
 cd /srv/salt/ && ln -sf example.top top.sls
-if [ ! -e /srv/pillar ]; then 
-    ln -sf /srv/salt/pillar/ /srv/pillar
-fi
+# this seems redundant wrt bootstrap.sh, which creates /srv/pillar?
```

Comment:
looks like it. please remove the commented out lines. these scripts have enough noise already

I did some shuffling until almost all of the bootstrap logic now lives in these three files (bootstrap, init-vagrant-minion, init-master) and the common stuff has been split out into update-master and highstate.